sdk/core/azure-core/tests/async_tests/test_basic_transport_async.py

```diff
@@ -989,3 +989,9 @@ def test_aiohttp_loop():
     loop = asyncio.get_event_loop()
     with pytest.raises(ValueError):
         transport = AioHttpTransport(loop=loop)
+
+def test_open_closed_aiohttp_transport():
+    transport = AioHttpTransport()
+    transport.close()
+    transport.open()
+    assert not transport.session
```

Comment:
What would this change mean for a client that attempts to send?
From what I can see in the transport code, if I attempted `transport.send()` after calling `transport.close()` (which I think is what a Client built on top of this would likely do) - then I think it will just raise an AttributeError on line 198.

Reply:
This is consistent with current behavior if user calls

with client:
     client.do_sth() 


# now transport is closed
client.do_sth_else()

right?